packages/polaris-viz/src/components/FunnelChartNext/components/Tooltip/Tooltip.scss

```diff
@@ -0,0 +1,33 @@
+.Rows {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 8px;
+}
+
+.Row {
+  font-size: 12px;
```

Comment:
Will we want this to be a different size on mobile?

Reply:
Is this changing?

Reply:
Based off the [Figma](https://www.figma.com/design/zYBKEXuelQ4eSSkyhuc9s7/%E2%9A%A1%EF%B8%8F-FUNNEL-UPDATES?node-id=638-18723&node-type=frame&t=XDK5v1ZQbvxQU3Pg-0) the tooltip size looks the same. I think Pablo has some thoughts but he is out. Just looking at some other charts, it doesn't look like font size change in tooltips on mobile, I can make a note about this and revisit with Pablo unless you have any strong opinions.